prql-compiler/src/sql/std.sql.prql

```diff
@@ -49,13 +49,13 @@ let read_csv = source -> s"read_csv_auto({source:0})"
 let mul = l r -> null
 
 #[binding_strength 100]
-let div_i = l r -> s"FLOOR(FLOOR(ABS({l:11} / {r:11})) * SIGN({l:11} / {r:11}))"
+let div_i = l r -> s"FLOOR(ABS({l:11} / {r:11})) * SIGN({l:0}) * SIGN({r:0})"
 
 #[binding_strength 11]
-let div_f = l r -> s"({l} * 1.0 / {r})"
+let div_f = l r -> s"({l:11} * 1.0 / {r:11})"
 
 #[binding_strength 11]
-let mod = l r -> s"{l} % {r}"
+let mod = l r -> s"{l:11} % {r:11}"
```

Comment:
When an interpolation does not have the binding strength specified it defaults to binding strength of the parent.

This was meant for simple cases like this one, where the whole s-string is just one binary operation.

Is behavior confusing? We can remove this defaulting and require all interpolations to have the strength specified. Or we can default to 100, which would insert parenthesis?

Reply:
Sorry for unnecessary change. I reverted it.

Reply:
No, no, I was asking for feedback on the new std.sql.prql format. It may have sounded passive aggressive, but it wasn't meant that way.

Reply:
Ah, thanks for the explanation.
I simply did not understand the mechanism when I made this change, but it certainly seems safe and reasonable to set the default to 100.
I do not have a strong opinion, however.